src/viewProviders/flinkArtifacts.test.ts

```diff
@@ -62,25 +62,21 @@ describe("FlinkArtifactsViewProvider", () => {
         getStubbedCCloudResourceLoader(sandbox);
 
       const mockArtifacts = [
-        new FlinkArtifact({
+        createFlinkArtifact({
           connectionId: resource.connectionId,
           connectionType: resource.connectionType,
           environmentId: resource.environmentId,
           id: "artifact1",
           name: "Test Artifact 1",
           description: "Test artifact description",
-          provider: "aws",
-          region: "us-east-1",
         }),
-        new FlinkArtifact({
+        createFlinkArtifact({
           connectionId: resource.connectionId,
           connectionType: resource.connectionType,
           environmentId: resource.environmentId,
           id: "artifact2",
           name: "Test Artifact 2",
           description: "Another test artifact",
-          provider: "aws",
-          region: "us-east-1",
         }),
```

Comment:
Having defaults allow us to reduce the amount of copy/paste boilerplate:
```suggestion
        createFlinkArtifact({
id: "artifact1",
name: "Test Artifact 1",
}),
        createFlinkArtifact({
id: "artifact2",
name: "Test Artifact 2",
}),
```